README.md

```diff
@@ -189,4 +180,14 @@ If you like this project please give it a star
 - I would like to thank microverse for their amazing school program
 - I would like to thank the code reviewer for reviewing my project
 - Watch🙏(https://www.loom.com/share/5cca07fe06174baab741632c0fedf1a9)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/citec-47/portfolio/blob/mobile-version/LICENSE.md)licensed.
```

Comment:
Well done on adding the `MIT` LICENSE to your README 📄 but the file you're referencing doesn't exist 🤒 which makes the link broken ❌  This is what you get when you try to go to the link 👇🏽
 
![Screenshot 2023-03-08 at 00 20 47](https://user-images.githubusercontent.com/45739963/223626252-f5362c02-7b5e-40e4-99cb-f0648db7684a.png)